spec/compiler/CodeSectionSpec.js

```diff
@@ -160,6 +160,18 @@ describe("CodeSection", function() {
       expect(section.hideColumns()).toEqual(true);
     });
   });
+  describe('removes uncorrectly escaped templates', function(){
+    it('contains a template', function() {
+      expect(section.cleanUpCode("[[<span class=\"hljs-attr\">.Disabled</span>]]")).toEqual("[[.Disabled]]")
+    });
+    it('does not alter valid escaped templates', function() {
+      expect(section.cleanUpCode("<span class=\"hljs-string\">\"[[.Timestamp]]\"</span>")).toEqual("<span class=\"hljs-string\">\"[[.Timestamp]]\"</span>")
+    });
+    it('does not alter valid escaped templates which contains if clause', function() {
+      expect(section.cleanUpCode("</span>[[if .PrevPageURL]]<span class=\"hljs-tag\">")).toEqual("</span>[[if .PrevPageURL]]<span class=\"hljs-tag\">")   
```

Comment:
Not sure this test makes sense. Much more interesting would be whether  escaped ifs, range & co work.